backend/coins/eth/account.go

```diff
@@ -50,7 +50,7 @@ type Account struct {
 	log *logrus.Entry
 }
 
-// NewAccount creats a new Account.
+// NewAccount creates a new account.
```

Comment:
`Account` was uppercase to match the struct name, but this does look nicer ;)

Reply:
Yeah, the PR was mainly intended to fix "creates". When I made the PR I realized that I could lowercase account in the same run and force pushed an amendment.